titus_isolate/crd/publish/kubernetes_predicted_usage_publisher.py

```diff
@@ -39,6 +40,7 @@ def publish(self):
         if len(self.__pod_manager.get_pods()) == 0:
             log.warning("No pods, skipping resource usage prediction")
             predictions = ResourceUsagePredictions({})
+            predictions.set_prediction_ts_ms(1000*int(time.time()))
```

Comment:
probably want `datetime.datetime.utcnow()` variant instead of time.time to avoid timezone consideration.